### DISCUSSION TITLE
포트폴리오 관련 무료 강의

### DISCUSSION THREAD

**Discussion by yeondooo** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#46] [General]
이미 보신 분들도 있겠지만, 
무료로 진행하는거고 시기가 프리온보딩 수료랑 맞물려서 같이 진행하면 좋을것 같아 공유합니다.
우아한형제들 프론트엔드 엔지니어 두분이 멘토님이네요!
링크: https://www.wanted.co.kr/events/udemykorea_resume?utm_source=wanted&utm_medium=share

**Comment by baechanghyeon** [comment]
오... 좋은 정보 감사합니다😄👊
